setup.json

```
@@ -91,7 +91,7 @@
         ],
         "notebook": [
             "jupyter~=1.0",
-            "notebook~=6.0"
+            "notebook~=6.0,>=6.1.5"
```

Technically valid, but I think this would be better:
```suggestion
            "notebook~=6.1,>=6.1.5"
```